arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -380,11 +381,16 @@
       "description": "The List operation response, that contains the database accounts and their properties."
     },
     "FailoverPolicies": {
-      "type": "array",
-      "description": "List of failover policies.",
-      "items": {
-        "$ref": "#/definitions/FailoverPolicy"
-      }
+      "properties": {
+        "failoverPolicies": {
```

Comment:
@amarzavery was not working before. It's a working change :)
https://github.com/Azure/azure-rest-api-specs/issues/701